components/ntp_background_images/browser/view_counter_service.cc

```diff
@@ -57,6 +57,7 @@ void ViewCounterService::RegisterProfilePrefs(
     user_prefs::PrefRegistrySyncable* registry) {
   registry->RegisterBooleanPref(
       prefs::kBrandedWallpaperNotificationDismissed, false);
+  registry->RegisterIntegerPref(prefs::kCountToBrandedWallpaper, 1);
```

Comment:
We could fetch the initial value from the model, @simonhong your thoughts?

Reply:
Yup, model could give initial value.